### PR TITLE
Add the option to skip commit

### PR DESCRIPTION
**Pull request by johno**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -70,6 +70,9 @@ function command(options) {
     cmd.push('echo "' + options.contents + '" >> ' + options.filename);
   }
 
-  cmd.push('git add .', 'git commit -m "' + options.message + '"');
+  if (!options.skipCommit) {
+    cmd.push('git add .', 'git commit -m "' + options.message + '"');
```
Comment (johno) [inline index.js:74]: I do wonder if maybe we want to split up the two commands since `git add .` doesn't exactly fall under `skipCommit`?

Comment (jonschlinkert) [reply index.js:74]: I trust your judgment, I'll add you to the repository so you can merge and publish when you feel like it's good. thanks!

Comment (jonschlinkert) [reply index.js:74]: done, added you to both this repository and as an owner on npm. thanks again!

Comment (johno) [reply index.js:74]: Thank you! 💟

Comment (jonschlinkert) [reply index.js:74]: you are very welcome!

**Pull request merged**